testing/xfpga/test_events_c.cpp

```diff
@@ -1066,8 +1154,295 @@ TEST_P(events_handle_p, irq_event_01) {
   ASSERT_EQ(FPGA_OK, sysfs_write_u64(path.c_str(), error_csr));
   EXPECT_EQ(FPGA_OK, xfpga_fpgaUnregisterEvent(handle_accel_, FPGA_EVENT_POWER_THERMAL, eh_));
 }
-
-//INSTANTIATE_TEST_CASE_P(events, events_handle_p,
-//                        ::testing::ValuesIn(test_platform::keys()));
 INSTANTIATE_TEST_CASE_P(events, events_handle_p,
                         ::testing::Values("skx-p-1s"));
+
+class mock_events_handle_p : public testing::TestWithParam<std::string> {
```

Comment:
These two classes are almost identical, the only difference being this one also opens the fme device. Can we merge these classes?